DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -249,6 +250,10 @@ public class DcContext {
         return DcArray(arrayPointer: dc_get_fresh_msgs(contextPointer))
     }
 
+    public func getFreshMessagesCount(chatId: Int) -> Int {
```

Comment:
there is already `getUnreadMessages` that does exactly the same under another name :)

so, for clarity, in this pr, i'd just this function - the things may get complicated enough :) is a subsequent pr, we can rename the function.

Reply:
Ups, I haven't seen that. Thanks! I'll fix that.